scripts/dev.js

```diff
@@ -38,11 +39,13 @@ const dev = async (maybeInit) => {
   const isInit = !fs.existsSync(path.join(TOOLBOX_ROOT, 'updateSchema.js')) || maybeInit
   const redis = new Redis(process.env.REDIS_URL)
   const toolboxPromise = compileToolbox()
+  exec('yarn graphql-codegen', (err) => {
```

Comment:
I wasn't too sure what the best approach would be here. While it works, it seems inconsistent with the rest of the file. Also, should this be included in the staging and prod builds too?

Reply:
dig into the graphql-codegen package & you should be able to figure out what function the bin is calling. then, you can call that directly

Reply:
updated!